pkg/driverbuilder/local.go

```diff
@@ -86,6 +135,20 @@ func (lbp *LocalBuildProcessor) Start(b *builder.Build) error {
 	// Fetch paths were kmod and probe will be built
 	srcModulePath := vv.GetModuleFullPath(c, kr)
 	srcProbePath := vv.GetProbeFullPath(c)
+
+	if len(lbp.srcDir) == 0 {
+		slog.Info("Downloading driver sources.")
```

Comment:
Downloading driver sources is now done in place using the new `LibsDownloadScript`, just like `docker` build processor does.